vignettes/articles/tbl_ard-functions.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: "Building Tables using tbl_ard*()"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+For most tables built using {gtsummary}, ARDs for quality control or table summaries can be extracted. However, there are instances where an ARD first approach is beneficial as building more complex tables may require analysis of the data structures prior to table construction. For this ARD first approach, {gtsummary} has `tbl_ard*()` functions that will generate a tables.
+
+-   `tbl_ard_continuous()` - for ARDs summarizing continuous variables.
+-   `tbl_ard_hierarchical()` - for ARDs containing nested or hierarchical data structures.
```

Comment:
Can you move `tbl_ard_hierarchical()` to the bottom? I can't think of a situation where this will be useful yet